src/main/java/net/mcreator/ui/dialogs/tools/MaterialPackMakerTool.java

```diff
@@ -61,23 +59,23 @@ private static void open(MCreator mcreator) {
 		color.setColor((Color) UIManager.get("MCreatorLAF.MAIN_TINT"));
 		name.enableRealtimeValidation();
 
-		props.add(new JLabel("Ore pack name:"));

```

Comment:
Please separate entries from the material pack class and the ore pack since they could be translated differently in different languages.

Reply:
Why? `Ore pack name` is still `Ore pack name` in every language, I don't understand...

Reply:
@Max094Reikeb Maybe @MysteriousWolf Wolf meant you made different pack makers use the same translation keys?

Reply:
These texts should be different, like Material pack here, but I never changed it. By giving different names here, one can change this, or at least in translations.

Reply:
@KlemenDEV oh okay. So I'll change it tho.

Reply:
It's done!